Gemfile.lock

```diff
@@ -305,7 +307,7 @@ DEPENDENCIES
   webmock (~> 3.0)
 
 RUBY VERSION
-   ruby 2.4.1p111
+ ruby 2.4.1p111
```

Comment:
These are still showing changes. What I want is to see _no_ change on lines 310 and 313 in the final diff.

You can check this locally by running `git diff master..head Gemfile.lock`; if you see any changes under RUBY VERSION or BUNDLED WITH, it's not the original spacing. You can also verify changes you've pushed to GitHub by looking at the [files tab](https://github.com/rubyforgood/playtime/pull/150/files); L310 and L313 should not be highlighted.

Reply:
Check d5970e2 👍